core/services/relay/evm/functions/config_poller.go

```diff
@@ -162,6 +162,9 @@ func (cp *configPoller) LatestConfig(ctx context.Context, changedInBlock uint64)
 	if err != nil {
 		return ocrtypes.ContractConfig{}, err
 	}
+	if len(lgs) == 0 {
+		return ocrtypes.ContractConfig{}, errors.New("no logs found")
```

Comment:
Does this always indicate an error or should we just warn here? For example, what if polling frequency is set to be more frequent than the rate of block production?